lib/gui/lvgl/Kconfig.graphical

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2018-2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2020 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Graphical settings"
+
+config LVGL_DISPLAY_DEV_NAME
+	string "Display device name"
+	default "DISPLAY"
+	help
+	  Name of the display device to use for rendering.
+
+config LVGL_HOR_RES
+	int "Horizontal Screen Resolution"
+	default 320
```

Comment:
Is this an intentional change (same for vertical res and dpi) from LVGL defaults?

Reply:
Ooops, good catch. I re-used last PR initial commit and missed that one.